decidim-assemblies/app/views/decidim/assemblies/assemblies/_promoted_assembly.html.erb

```diff
@@ -1,27 +1 @@
-<div class="card card--full card--process">
-  <div class="row collapse card--process__row">
-    <div class="columns mediumlarge-8 large-6 card--process__column">
-      <div class="card__content">
-        <%= link_to assembly_path(promoted_assembly), class: "card__link" do %>
-          <h2 class="card__title"><%= translated_attribute promoted_assembly.title %></h2>
-        <% end %>
-        <%= decidim_sanitize_editor html_truncate(translated_attribute(promoted_assembly.short_description), length: 630, separator: "...") %>
-        <%= link_to assembly_path(promoted_assembly), class: "button small hollow card__button" do %>
-          <%= t("assemblies.promoted_assembly.more_info", scope: "layouts.decidim") %>
-        <% end %>
-      </div>
-    </div>
-    <div class="columns mediumlarge-8 large-6 card--process__column">
-      <div class="card--full__image"
-           style="background-image:url('<%= promoted_assembly.attached_uploader(:hero_image).path %>')">
-        <div class="card__content row collapse">
-          <div class="large-6 large-offset-6 columns">
-            <%= link_to assembly_path(promoted_assembly), class: "button expanded button--sc" do %>
-              <%= t(promoted_assembly.cta_button_text_key, scope: "layouts.decidim.assemblies.promoted_assembly") %>
-            <% end %>
-          </div>
-        </div>
-      </div>
-    </div>
-  </div>
-</div>
+<%= card_for promoted_assembly, highlight: true, size: :g %>
```

Comment:
Why do you keep the partial? This is being used only in old redesign in : `decidim/decidim-assemblies/app/views/decidim/assemblies/assemblies/index.html.erb:15`

Reply:
Moved to the view directly